Dockerfile

```
@@ -0,0 +1,19 @@
+FROM ubuntu:22.04
+
+COPY inittimezone /usr/local/bin/inittimezone
+RUN chmod +x inittimezone.
```

```suggestion
RUN chmod +x /usr/local/inittimezone
```
but you are actually not using this file?